iib/workers/tasks/git_utils.py

```diff
@@ -0,0 +1,249 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+"""This file contains functions for saving changes to GitLab."""
```

Comment:
With @lipoja we agreed that the code should not be Gitlab specific (after we merged the `iib_index_configs_gitlab_tokens_map` change). Would it be possible to change the terminology from "Gitlab" to "Git" where it is possible?